### PR TITLE
fix(region selector): moving selector when in tiny viewport

### PR DESCRIPTION
**Pull request by runrog**

### CODE REVIEW

File: styleguide/derek/global-components/modal/_region-select.scss [hunk 1]
```diff
@@ -128,3 +128,27 @@
     column-gap: 15px;
   }
 }
+
+@media only screen and (max-width: $screen-xs-min - 1) {
```
Comment (WidgetsBurritos) [inline styleguide/derek/global-components/modal/_region-select.scss]: I don't think we want to stop at 479px here.   Some android devices use 480px and there's still some overlap there: 
image

I'd say either set it to `$screen-xs-min` (480px) or `$screen-xs-max` (767px).

File: styleguide/derek/global-components/modal/_region-select.scss [hunk 2]
```diff
@@ -128,3 +128,9 @@
     column-gap: 15px;
   }
 }
+
+@media only screen and (max-width: $screen-xs-max) {
+  .basement-container {
+    padding-bottom: 35px;
```
Comment (WidgetsBurritos) [inline styleguide/derek/global-components/modal/_region-select.scss]: So looking at production right now the chat button takes up a good 65px or so.  So maybe we just bump the padding to 70px instead?

**Comment by xinbin** [comment]
looks good to me
Screen Shot 2019-06-28 at 12 35 26 PM

File: styleguide/derek/global-components/modal/_region-select.scss (hunk 2 above)
Comment (pobtastic) [reply styleguide/derek/global-components/modal/_region-select.scss]: He made it 75px ~ and I think this looks fine.

**Pull request merged**